lib/router/router.js

```diff
@@ -109,6 +126,17 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && ((this.options.ifNoOperation && this.options.ifNoOperation.justForward === true) || routeInfo.urlInfo.query.noop === 'true')) {
```

Comment:
It is not applying to all resources. It just enables to apply noop=true to all requests where no operation parameter is added. It still process the operations if operation parameters are added to request url.
I am going to explain my scenario, So you can understand my need.
I have huge jpg images where the filename is a number without extension.
90% of request are for a few small resized versions. So I add operation parameter in request url to get it.
Sometimes I need to get the huge original. So I use a request url without any parameter.
But this way the image is processed and I get a huge unnecessary image in cache.
The noop could work but as my images filenames have no file extension I can not define a content-type for it. I could change to set no content-type header and trust the browser for it, but it do not sounds good.
Enabling it as an option enables to set a content-type if no file extension is found and enable to make the noop=true as the default behavior for the running instance if desired. In my case it is desirable in my running instance sine I do not want to trust endpoint user to provide the noop query to save my server resources.